spec/controllers/queue_items_controller_spec.rb

```diff
@@ -90,11 +80,122 @@
         expect(QueueItem.count).to eq(1)
       end
 
+      it "normalizes the remaining queue items" do
+        queue_item1 = Fabricate(:queue_item, user: user, order:1)
+        queue_item2 = Fabricate(:queue_item, user: user, order:2)
+        delete :destroy, id: queue_item1.id
+        expect(queue_item2.reload.order).to eq(1)
+      end
+
       it "redirects to my queue path" do
         queue_item = Fabricate(:queue_item, user: user)
         delete :destroy, id: queue_item.id
         expect(response).to redirect_to my_queue_path
       end
     end
+    describe "POST #update_queue" do
+      it "deletes a rating if input is set to blank" do
```

Comment:
What is this spec about? it has no setup, expectation. etc. Delete it if its not doing anything for you.